Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.9
+RUN apk add --no-cache python && \
+    python -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip install --upgrade pip setuptools && \
+    rm -r /root/.cache
+
+WORKDIR /presence-analyzer-mgotkowski
```

Comment:
could be `/code`.
In general if the project name be shorter it could possible fit as is but remember that this basically generates `/NAME` in the root filesystem of the container.
So inside there are normal linux dirs like
`/var`
`/usr`
`/root`
`/home`
and there bumps the
`/presence-analyzer-mgotkowski`
a `/code` or `/src` would fit better ;)